javascript-functions/main.js

```diff
@@ -0,0 +1,41 @@
+function convertMinutesToSeconds(minutes) {
+  const i = minutes * 60;
+
+  return i;
+}
+
+console.log('The amount of seconds is:', convertMinutesToSeconds(5));
+
+function greetPerson(name) {
+  const greeting = 'Hey, ' + name;
+
+  return greeting;
+}
+
+console.log(greetPerson('Beavis'));
+
+function getArea(width, height) {
+  const i = width * height;
+
+  return i;
+}
+
+console.log('The area is: ' + getArea(17, 42));
+
+function getFirstName(person) {
+  const name = { firstName: 'Lelouche', lastName: 'Lamperouge' };
```

Comment:
Rather than creating the object inside of the function, you should be creating the object outside the function, and passsing in the object via your `person` parameter